sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -1557,6 +1557,19 @@ class PlanResolutionSuite extends AnalysisTest {
     checkFailure("testcat.tab", "foo")
   }
 
+  test("SPARK-32356: forbid null type in create view") {
```

Comment:
can we move this to `SQLViewSuite` as well?

Reply:
Moved.